include/tinyalsa/mixer.h

```diff
@@ -93,6 +93,8 @@ int mixer_add_new_ctls(struct mixer *mixer);
 
 const char *mixer_get_name(const struct mixer *mixer);
 
+unsigned int mixer_ctl_get_dev_num(const struct mixer_ctl *ctl);
```

Comment:
1. Please move this function declaration to the lines between 120 and 160.
2. Please rename to `mixer_ctl_get_device` to match the naming rule.

Reply:
Thanks for the review. Updated changes in pull request.